src/main/java/com/liferay/faces/util/application/internal/ResourceValidatorImpl.java

```diff
@@ -83,6 +79,25 @@ public ResourceValidatorImpl(ArrayList<Pattern> excludeResourcePatterns,
 		}
 	}
 
+	private static List<String> getViewMappings(ExternalContext externalContext, String... viewMappingsParamNames) {
```

Comment:
@stiemannkj1 Not sure of the necessity to have a separate method named `getViewMappings` since it is only called from one place. Is there a reason why it shouldn't have been kept inline?